geometry_msgs/PoseStamped start
geometry_msgs/PoseStamped goal
---
nav_msgs/Path path
nav_msgs/Path horiz_path